include/pmacc/PMaccConfig.cmake

```diff
@@ -320,19 +320,15 @@ endif()
 message(STATUS "Boost: result_of with TR1 style and decltype fallback")
 set(PMacc_DEFINITIONS ${PMacc_DEFINITIONS} -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK)
 
-# Boost >= 1.60.0 and CUDA != 7.5 failed when used with C++11
-# seen with Boost 1.60.0 - 1.62.0 (maybe later) and CUDA 7.0 and 8.0
-# CUDA 7.5 works without a workaround
-# CUDA 8.0.44 bug reappeared, but work-around was mainlined in 1.63.0+
-# CUDA 9.0 unknown, likely fixed
-# CUDA 9.1.85-3 and above fixed
+# Boost >= 1.60.0 and CUDA 8.0 fail to compile on mp_defer
+# seen with Boost 1.60.0 - 1.63.0 (latter got config work-around below)
+# CUDA 9.0+ fixed
 if("${ALPAKA_CUDA_COMPILER}" STREQUAL "nvcc")
     if( (Boost_VERSION GREATER 105999) AND
-        (NOT CUDA_VERSION VERSION_EQUAL 7.5) AND
-        (CUDA_VERSION VERSION_LESS 9.1) )
+        (CUDA_VERSION VERSION_EQUAL 8.0) )
         # Boost Bug https://svn.boost.org/trac/boost/ticket/11897
-        message(STATUS "Boost: Disable template aliases")
-        set(PMacc_DEFINITIONS ${PMacc_DEFINITIONS} -DBOOST_NO_CXX11_TEMPLATE_ALIASES)
+        message(STATUS "Boost: Disable variadic templates")
```

Comment:
Actually disabling variadic templates was the original work-around to address the issues in `mp_defer`.
Just by coincidence, `mp_defer` is also no included when setting the totally unrelated `BOOST_NO_CXX11_TEMPLATE_ALIASES` switch, introduced here while trying to add CUDA 8.0 support:
https://github.com/ComputationalRadiationPhysics/picongpu/commit/3c40840d99b2493a9a5f2ae011a2003ba226893a